apps/game/src/render/mod.rs

```diff
@@ -0,0 +1,90 @@
+use geom::{Distance, Pt2D, Tessellation};
+use map_gui::colors::ColorScheme;
+use map_gui::render::DrawOptions;
+use map_gui::AppLike;
+use map_model::{Map, NORMAL_LANE_THICKNESS, SIDEWALK_THICKNESS};
+use sim::{DrawCarInput, PersonID, Sim, VehicleType};
+use widgetry::{Color, GfxCtx, Prerender};
+
+pub use crate::render::agents::{AgentCache, UnzoomedAgents};
+use crate::render::bike::DrawBike;
+use crate::render::car::DrawCar;
+pub use crate::render::pedestrian::{DrawPedCrowd, DrawPedestrian};
+use crate::ID;
+
+mod agents;
+mod bike;
+mod car;
+mod pedestrian;
+
+// Like map_gui's Renderable, but uses our ID type
+pub trait GameRenderable {
+    // TODO This is expensive for the PedCrowd case. :( Returning a borrow is awkward, because most
+    // Renderables are better off storing the inner ID directly.
+    fn get_id(&self) -> ID;
+    fn draw(&self, g: &mut GfxCtx, app: &dyn AppLike, opts: &DrawOptions);
+    // Higher z-ordered objects are drawn later. Default to low so roads at -1 don't vanish.
+    fn get_zorder(&self) -> isize {
+        -5
+    }
+    // This outline is drawn over the base object to show that it's selected. It also represents
+    // the boundaries for quadtrees. This isn't called often; don't worry about caching.
+    fn get_outline(&self, map: &Map) -> Tessellation;
+    fn contains_pt(&self, pt: Pt2D, map: &Map) -> bool;
+}
+
+impl<R: map_gui::render::Renderable> GameRenderable for R {
```

Comment:
Bit of magic here -- any static map_gui renderable also satisfies the new renderable trait. the ID mapping is the only difference